parts/k8s/cloud-init/jumpboxcustomdata.yml

```diff
@@ -7,20 +7,20 @@ write_files:
   encoding: gzip
   owner: "root"
   content: !!binary |
-    {{CloudInitData "provisionSource"}}
+    {{ CloudInitData "provisionSource" }}
```

Comment:
What's the purpose of adding extra whitespace after and before the `{{` and `}}`?

Reply:
It's to be coherent with the `{{- ... -}}` syntax which can not suffer to not have whitespaces after or before the `-`.